scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/LazyTokenIterator.scala

```diff
@@ -133,6 +133,10 @@ private[parsers] class LazyTokenIterator private (
       foundIndentation
   }
 
+  def previousIndentation: Int = {
+    sepRegions.lift(1).fold(0)(_.indent)
```

Comment:
what's lift(1)? is it headOption?

Reply:
it takes the second element of the list if it exists